.github/workflows/build-dev.yml

```diff
@@ -41,9 +44,12 @@ jobs:
         TAG=$(date +%Y-%m-%d)-$(git rev-parse --abbrev-ref HEAD)-$(git rev-parse --short HEAD)
         echo "ecw5410-$TAG" > feeds/wlan-ap/opensync/src/vendor/tip/.pkgname
         make TARGET=ECW5410
-        TAR_NAME="ecw5410-$TAG.tar.gz"
+        TAR_NAME="ecw5410-artifacts-$TAG.tar.gz"
         tar cfz "$TAR_NAME" -C openwrt/bin/targets/ipq806x/generic .
-        curl -u "$GH_BUILD_USERNAME":"$GH_BUILD_PASSWORD" -T "$TAR_NAME" "https://tip.jfrog.io/artifactory/tip-wlan-ap-firmware/ecw5410/dev/"$TAR_NAME""
+        curl -u "$GH_BUILD_USERNAME":"$GH_BUILD_PASSWORD" -T "$TAR_NAME" "https://tip.jfrog.io/artifactory/tip-wlan-ap-firmware/ecw5410/dev/artifacts/"$TAR_NAME""
+        UPGRADE_TAR_NAME="ecw5410-$TAG.tar.gz"
+        tar cfz "$UPGRADE_TAR_NAME" -C openwrt/bin/targets/ipq806x/generic $(find openwrt/bin/targets/ipq806x/generic -type f \( -name '*sysupgrade*' -o -name 'sha256sums' \) -printf "%f\n")
```

Comment:
wont this also include the factory images ? we might want to filter them out ...

Reply:
Generating two tar files here.  One with sysupgrade image and sha256sum file.  The other with all the build artifacts (.ipk files, etc) which will also include the factory image.